### PR TITLE
Update the last community update in scalability report

### PR DESCRIPTION
**Pull request by mm4tt**

### CODE REVIEW

File: sig-scalability/annual-report-2021.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ There isn't any formal process. Updates are given during regular SIG meetings.
 
 **When was your last monthly community-wide update? (provide link to deck and/or recording)**
 
-January'20: [slide deck]
+August'20: [slide deck]
```
Comment (wojtek-t) [inline sig-scalability/annual-report-2021.md:36]: I knew I was missing somthing but I couldn't find these slides...

How about putting both here instead?

Comment (mm4tt) [reply sig-scalability/annual-report-2021.md:36]: Initially, I wanted to do that, but then I read the heading - "last ... community-wide update" (btw, no idea why there is monthly there). 
But I'm fine with having both as well. You decide :)

Comment (wojtek-t) [reply sig-scalability/annual-report-2021.md:36]: @mrbobbytables - thoughts? [I would keep both, because they were both in 2020]

Comment (mrbobbytables) [reply sig-scalability/annual-report-2021.md:36]: I'd lean towards both, it's still useful information. :)

Comment (mm4tt) [reply sig-scalability/annual-report-2021.md]: Done. Sorry, I completely forgot about it.

**Comment by wojtek-t** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**